src/Result.ts

```diff
@@ -18,11 +17,9 @@ export type Result = {
 const defaultDisabledRules = ['bypass', 'landmark-one-main', 'page-has-heading-one', 'region'];
 
 /**
- * Run Axe on a browser page for a story.
+ * Run Axe on a browser page that is displaying a story.
  */
 export async function fromPage(page: Page, story: ProcessedStory): Promise<Result> {
-  await showStory(page, story);
```

Comment:
`Result.fromPage` will now only work on a page that is **_already_** showing a particular story. TestBrowser now handles that.

I think this is okay. But who knows.